src/parser/src/main/java/io/github/syst3ms/skriptparser/expressions/ExprSingleParameter.java

```diff
@@ -0,0 +1,45 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.lambda.ArgumentSection;
+import io.github.syst3ms.skriptparser.lang.lambda.SectionValue;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+
+public class ExprSingleParameter extends SectionValue<ArgumentSection, Object> {
+
+    static {
+        Main.getMainRegistration().addExpression(
+                ExprSingleParameter.class,
+                Object.class,
+                true,
+                "[the] (input|value|parameter)"
```

Comment:
I still am advocating for the removal of 'value'. It is way too common and generic and can clutter future syntax. If this particular option stays, consider lowering the priority.
'Parameter' and 'input' describe their purpose immediately, while 'value' as an option is vague and unclear.